flang/include/flang/Frontend/CodeGenOptions.h

```diff
@@ -49,6 +49,26 @@ class CodeGenOptionsBase {
 class CodeGenOptions : public CodeGenOptionsBase {
 
 public:
+  /// The type of frame pointer used
+  enum class FramePointerKind {
+    None,    // Omit all frame pointers.
+    NonLeaf, // Keep non-leaf frame pointers.
+    All,     // Keep all frame pointers.
+  };
+
+  static llvm::StringRef getFramePointerKindName(FramePointerKind Kind) {
+    switch (Kind) {
+    case FramePointerKind::None:
+      return "none";
+    case FramePointerKind::NonLeaf:
+      return "non-leaf";
+    case FramePointerKind::All:
+      return "all";
+    }
+
+    llvm_unreachable("invalid FramePointerKind");
+  };
+
```

Comment:
Move this code and the same code in `clang/include/clang/Basic/CodeGenOptions.h` to llvm/include/llvm/Frontend/Driver.

Reply:
Issue has been addressed